app/routines/transfer_signup_state.rb

```diff
@@ -4,9 +4,9 @@ class TransferSignupState
 
   def exec(signup_state:, user:)
     fatal_error(code: :no_signup_email) if signup_state.nil?
-    fatal_error(code: :not_verified) if !signup_state.verified?
 
-    run(AddEmailToUser, signup_state.contact_info_value, user, {already_verified: true})
+    run(AddEmailToUser, signup_state.contact_info_value, user, {already_verified: signup_state.verified?})
```

Comment:
I know we don't have verified emails now...but this is old code... wondering why it is ok to now to have unverified emails transferred to the user... when from way back (when we didn't have trusted data) we only transferred verified emails?

Reply:
Hm, well we could force the user through verification but that seems like it'd be cumbersome, especially since the FE will now prompt the user to verify the email using the navbar nags.

Seems to me like it's best to just transfer what we've got and let the UI verify as needed

Reply:
ok, I will ponder this in the AM.